test/components/remote_retro_test.js

```diff
@@ -32,4 +39,28 @@ describe("RemoteRetro component", () => {
       expect(hotjarSpy).calledWith("trigger", CLOSED)
     })
   })
+
+  describe("when component updates with a new facilitator", () => {
+    let newFacilitator
+    beforeEach(() => {
+      newFacilitator = {
+        ...stubUser,
+        is_facilitator: true,
+      }
+    })
+
+    it("calls the new facilitator action", () => {
+      const wrapper = shallow(<RemoteRetro {...defaultProps} />)
+      wrapper.setProps({ user: newFacilitator })
+      expect(defaultProps.actions.newFacilitator.calledWith(defaultProps.retro))
+    })
+  })
+
+  describe("when component updates new stage", () => {
```

Comment:
per my comment below, I think we'll want to update the `describe` language here to reflect that the conditions we're setting up is a non-change to the current user's facilitatorship

Reply:
agreed